test/sa_construct_test.cpp

```diff
@@ -40,7 +40,12 @@ TEST_F(sa_construct_test, divsufsort)
     construct_sa<8>(config);
     memory_monitor::stop();
     {
+        #if SDSL_HAS_CEREAL
```

Comment:
Please remove the indention for the `if` in this file as you did in the other files.